src/axom/klee/tests/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#------------------------------------------------------------------------------
+# Unit tests for Klee component
+#------------------------------------------------------------------------------
+
+#
+# Specify list of tests
+#
+
+set(gtest_klee_tests
+    klee_config.cpp
+    klee_geometry.cpp
+    klee_geometry_operators.cpp
+    klee_geometry_operators_io.cpp
+    klee_io.cpp
+    klee_io_util.cpp
+    klee_shape.cpp
+    klee_shape_set.cpp
+    klee_units.cpp
+   )
+
+blt_add_library(
+        NAME        klee_test_utils
+        SOURCES     KleeTestUtils.cpp
+        HEADERS     KleeTestUtils.hpp KleeMatchers.hpp
+        DEPENDS_ON  klee gtest gmock
+        FOLDER      axom/klee/tests
+        OBJECT      TRUE
+        )
+
+#
+# Add gtest based tests
+#
+foreach(test ${gtest_klee_tests})
+    get_filename_component( test_name ${test} NAME_WE )
+    blt_add_executable( NAME ${test_name}_test
+                        SOURCES ${test}
+                        OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
+                        DEPENDS_ON axom gtest gmock klee_test_utils
+                        FOLDER      axom/klee/tests )
+    blt_add_test( NAME ${test_name}
+                  COMMAND ${test_name}_test )
+endforeach()
```

Comment:
Also ran into an issue when compiling `klee` tests for config `host-configs/rzansel-klee-clang@8.0.1_nvcc_xlf.cmake` (nvcc multiple definitions error).
[Mirroring Inlet tests' dependencies](https://github.com/LLNL/axom/blob/fd9d96919df1f5b94ff4ecab4b38c748c8933bc9/src/axom/inlet/tests/CMakeLists.txt#L24-L38) resolved it for me.
```suggestion
blt_add_library(
        NAME        klee_test_utils
        SOURCES     KleeTestUtils.cpp
        HEADERS     KleeTestUtils.hpp KleeMatchers.hpp
        DEPENDS_ON  axom gtest gmock
        FOLDER      axom/klee/tests)

#
# Add gtest based tests
#
foreach(test ${gtest_klee_tests})
    get_filename_component( test_name ${test} NAME_WE )
    blt_add_executable( NAME ${test_name}_test
                        SOURCES ${test}
                        OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
                        DEPENDS_ON gtest klee_test_utils
                        FOLDER      axom/klee/tests )
    blt_add_test( NAME ${test_name}
                  COMMAND ${test_name}_test )
endforeach()
```